src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -117,7 +117,6 @@ void publish_commit_completion(rados_completion_t completion, void *arg) {
 };
 
 class Manager : public DoutPrefixProvider {
-  const size_t max_queue_size;
```

Comment:
how is this change related?

Reply:
that member variable is unused after add_persistent_topic() was removed from the class